src/components/basic/display/MarketPrice/index.tsx

```diff
@@ -12,7 +12,7 @@ export interface Props {
   setError?: (msg?: string) => void;
 }
 
-function component(props: Props): JSX.Element {
+function UMarketPrice(props: Props): JSX.Element {
```

Comment:
Why the `U` prefix?

Reply:
U stands for Unsafe or Unmemorized. `component` is a name that causes a ton of issues with typescript and react.

Reply:
As discussed over slack, I'll remove the intermediate component and apply `memo` directly.